CHANGELOG.md

```diff
@@ -6,14 +6,30 @@ to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 ### Added
+- A banner has been added to the homepage to promote Maker Party,
+  a Mozilla global campaign.
+- Each section in the Web Literacy Map can now be visited via a
+  unique permalink. Clicking on the section sign (§) at the top-right
+  of each expander section will set browser's current URL to a
+  permalink for the section. Visiting a section directly through its
+  permalink will also automatically expand that section to attract
+  users' attention.
+- Tools page has been added(https://teach.mozilla.org/tools). Currently
+  there's no UI element on the site that leads to that page. However,
+  a menu item "Tools" will be added to the sidebar in the next release.
```

Comment:
Can you write "a follow-up release" rather than "the next release", since we can't actually guarantee that the next release will contain it? (For instance, suppose we push a release tomorrow that only has one or two critical fixes in it, etc.)